src/lib/metric_ingest.py

```diff
@@ -206,20 +208,34 @@ def extract_typed_value_key(time_serie):
     return typed_value_key
 
 
-def create_dimensions(time_serie: Dict) -> List[DimensionValue]:
+def create_dimension(name: str, value: str, context: LoggingContext = LoggingContext(None)) -> DimensionValue:
+    if len(name) > MAX_DIMENSION_NAME_LENGTH:
+        context.log("MINT rejects dimension names longer that {max_dim_name_length} chars. Dimension name \"{name}\" "
```

Comment:
[f-formatted strings](https://docs.python.org/3/tutorial/inputoutput.html) are generally used in this project, because these are more readable than `format` call